sections/footer.liquid

```diff
@@ -37,150 +37,163 @@
 {%- endstyle -%}
 
 <footer class="footer color-{{ section.settings.color_scheme }} gradient section-{{ section.id }}-padding">
+  {%-liquid 
+    assign no_social_icons = false
+    if settings.social_facebook_link == blank and settings.social_instagram_link == blank and settings.social_youtube_link == blank and settings.social_tiktok_link == blank and settings.social_twitter_link == blank and settings.social_pinterest_link == blank and settings.social_snapchat_link == blank and settings.social_tumblr_link == blank and settings.social_vimeo_link == blank
+      assign no_social_icons = true
+    endif
+
+    assign only_empty_brand = false
+    if section.blocks.size == 1 and section.blocks[0].type == 'brand_information' and no_social_icons and settings.brand_image == blank and settings.brand_headline == blank and settings.brand_description == blank and section.settings.newsletter_enable == false and section.settings.enable_follow_on_shop == false
```

Comment:
These are very ugly lines of code, I know. 

However, it's the only short-term solution I was able to find. We can't use `:has` yet (not supported in Firefox) and a complete rewrite of footer, possibly including visual changes, would be required to achieve the same result. We'll kick off this discussion in a separate issue.

Reply:
oki